src/Files.Uwp/ViewModels/ColumnsViewModel.cs

```diff
@@ -129,28 +128,59 @@ private void SetColumnSizeProportionally(double factor)
             SizeColumn.TryMultiplySize(factor);
             StatusColumn.TryMultiplySize(factor);
         }
+
+        public override bool Equals(object obj)
+        {
+            if (obj == null)
+            {
+                return false;
+            }
+            if (obj == this)
+            {
+                return true;
+            }
+            if (obj is ColumnsViewModel model)
+            {
+                return (
+                    model.DateCreatedColumn.Equals(this.DateCreatedColumn) &&
+                    model.DateDeletedColumn.Equals(this.DateDeletedColumn) &&
+                    model.DateModifiedColumn.Equals(this.DateModifiedColumn) &&
+                    model.ItemTypeColumn.Equals(this.ItemTypeColumn) &&
+                    model.NameColumn.Equals(this.NameColumn) &&
+                    model.OriginalPathColumn.Equals(this.OriginalPathColumn) &&
+                    model.SizeColumn.Equals(this.SizeColumn) &&
+                    model.StatusColumn.Equals(this.StatusColumn) &&
+                    model.TagColumn.Equals(this.TagColumn));
+            }
+            return base.Equals(obj);
+        }
+
+        public override int GetHashCode()
+        {
+            return base.GetHashCode();
+        }
```

Comment:
As mentioned earlier, do we need this and if so, equals-hashcode contract might cause issues if the DB relies on it.

Reply:
Keeping comment below.